Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-filter/ContestFilter.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo, useState } from 'react';
+
+import Heading, { HeadingType } from '../../guidelines/headings/Heading';
+import List, { Orientation } from '../../guidelines/lists/List';
+import { FilterType, IFilter } from '../../../common/contest-types';
+import ExpandButton from '../../guidelines/buttons/ExpandButton';
+import Button, { ButtonSize, ButtonType } from '../../guidelines/buttons/Button';
+import concatClassNames from '../../../utils/class-names';
+
+import styles from './ContestFilter.module.scss';
+
+
+interface IContestFilterTypeProps {
+    values: IFilter[];
+    type: FilterType;
+    orientation?: Orientation;
+    onFilterClick: (filterId: number) => void;
+}
+
+const ContestFilter = ({
+    values,
+    type,
+    orientation,
+    onFilterClick,
```

Comment:
I would name this `onSelect`. We are in the `ContestFilter` component, no need for the explicity here

Reply:
Done.